localization/tsconfig.json

```diff
@@ -5,5 +5,5 @@
         "experimentalDecorators": true,
         "resolveJsonModule": true
     },
-    "include": ["**/*.ts", "**/*.tsx", "**/*.json"]
+    "include": ["**/*.ts", "**/*.tsx", "**/*.json", "src/DayJSConfig.js"]
```

Comment:
Oh, let's switch `DayJSConfig.js` to `DayJSConfig.ts` instead.... It looks like @BigRoma has forgotten to do this